cmd/explaintest/t/access_path_selection.test

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE `pay_daily` (
+`id` varchar(50) NOT NULL DEFAULT '',
+`a_id` bigint(11) NOT NULL DEFAULT '0',
+`b_id` bigint(11) NOT NULL DEFAULT '0',
+`time_stamp` bigint(11) DEFAULT NULL,
+`date_key` int(20) DEFAULT NULL,
+`hour` int(11) DEFAULT NULL,
+`m_type` int(11) DEFAULT NULL,
+`c_id` bigint(11) DEFAULT NULL,
+`c_name` varchar(50) DEFAULT NULL,
+`d_id` bigint(11) DEFAULT NULL,
+`d_name` varchar(32) DEFAULT NULL,
+`is_custom` varchar(32) DEFAULT NULL,
+`io_type` int(11) DEFAULT NULL,
+`p_cnt` bigint(11) DEFAULT NULL,
+`p_amt` bigint(11) DEFAULT NULL,
+`update_source` int(11) DEFAULT NULL,
+`update_time` timestamp DEFAULT NULL,
```

Comment:
Can we remove the useless columns?

Reply:
of course.
All useless columns have been removed now.